.eslintrc.js

```diff
@@ -1,11 +1,25 @@
-
-/**
- * These rules enforce the Hack Reactor Style Guide
- *
- * Visit this repo for more information:
- *   https://github.com/reactorcore/eslint-config-hackreactor
- */
-
 module.exports = {
-    extends: './node_modules/eslint-config-hackreactor/index.js'
-  };
\ No newline at end of file
+  env: {
```

Comment:
As an FYI for the future, if you want your EsLint to also work with Jest, you'll have to add to this configuration.

Reply:
cool added jest onto the list.